### ISSUE TITLE
Process for reviewing new projects added to Giveth - marking them as listed/unlisted

### ISSUE THREAD

**Issue by laurenluz** [Giveth/giveth-planning#379]
- should add default where new projects are automatically listed after 2 weeks @laurenluz to make issue in Giveth Next
- @laurenluz to checkin w Mateo to determine when admin dashboard will be ready

- verification team can be part of listing/unlisting projects after dashboard

- until dashboard is up, need to go into the database to determine which projects are "new" in the interim @divine-comedian

**Comment by divine-comedian** [comment]
I tagged Donna in this issue so she could lend us a hand keeping track of new projects until we have implemented an automated system

this is the current spreadsheet we are using @DonnaOftadeh

**Comment by laurenluz** [comment]
Dashboard is still on staging, waiting for it to go live.

conclusion for listed unlisted... projects are default unlisted and go listed after 3? weeks of inactivity

**Comment by laurenluz** [comment]
@divine-comedian to champion! push to a close

**Comment by laurenluz** [comment]
@WhyldWanderer feels strongly that automatic trigger is wway too much overhead for the value add

**Comment by divine-comedian** [comment]
Update from development that I overheard in the dev sync - the admin dashboard is live on production, just need to secure the access credentials .. will follow up in discord

**Comment by laurenluz** [comment]
Ashley currently watching log/event channel in discord and reviewing projects when new ones are added

Admin dashboard is not live yet. "Figuring out the flow" - now we added "cancelled* and we're still adding the quasi status of "reviewed/not reviewed"

@ForestSoleil to work on system for organizing team around unlisted/listed verifying/unverified

Blocked by development of dashboard currently.

Ashley to send forest login credentials

**Comment by WhyldWanderer** [comment]
The automated unlisted to listed thing has been adjusted to two weeks.
We can filter by 'not reviewed' and list/unlist projects as they come in and I will incorporate this in our biweekly verification parties to ensure that all projects get reviewed in a timely manner.

**Comment by WhyldWanderer** [comment]
Im going to close this for now and if we need to revisit this again we can make a new issue
